setup.py

```diff
@@ -86,6 +112,13 @@ def run(self):
         dotnet_modules = [lib for lib in orig_modules if isinstance(lib, DotnetLib)]
         other_modules = [lib for lib in orig_modules if not isinstance(lib, DotnetLib)]
 
+        if dotnet_modules:
+            if os.path.isfile(CONFIGURED_PROPS):
+                self.announce("Already configured", level=distutils.log.INFO)
+            else:
+                self.announce("Writing configured.props...", level=distutils.log.INFO)
+                _write_configure_props()
```

Comment:
Is this taking a lot of time? I feel like check for existing file might cause surprising behavior during repeated builds on the same machine.

Reply:
It's not, I just want to add configuration options that I don't want to have to repeat all the time. If those are not needed, this step can also be simplified.

Reply:
Ah, now I understand that comment. Yeah, we should just always do this step. I'm going to work more on our build system once this lands.